documentation/Functions.md

```diff
@@ -1,290 +1,187 @@
 ## Functions 
 The following functions are already implemented:
-* [installDevonfwIde](#installDevonfwIde)
-* [restoreDevonfwIde](#restoreDevonfwIde)
-* [restoreWorkspace](#restoreWorkspace)
-* [changeWorkspace](#changeWorkspace)
-* [executeCommand](#executeCommand)
-* [installCobiGen](#installCobiGen)
-* [cobiGenJava](#cobiGenJava)
-* [createDevon4jProject](#createDevon4jProject)
-* [buildJava](#buildJava)
-* [createFile](#createFile)
-* [changeFile](#changeFile)
-* [createFolder](#createFolder)
-* [cloneRepository](#cloneRepository)
-* [runServerJava](#runServerJava)
-* [buildNg](#buildNg)
-* [npmInstall](#npmInstall)
-* [dockerCompose](#dockerCompose)
-* [downloadFile](#downloadFile)
-* [displayContent](#displayContent)
-* [adaptTemplatesCobiGen](#adaptTemplatesCobiGen)
-* [createDevon4ngProject](#createDevon4ngProject)
-* [addSetupScript](#addSetupScript)
+ * [CobiGen Functions](#cobiGen)
+    1. [adaptTemplatesCobiGen](#adaptTemplatesCobiGen)
+    2. [cobiGenJava](#cobiGenJava)
+    3. [installCobiGen](#installCobiGen) 
+
+* [devonfw-ide Functions](#ideSpecific)
+    1. [installDevonfwIde](#installDevonfwIde)
+    2. [restoreDevonfwIde](#restoreDevonfwIde)
+
+* [devon4j Functions](#devon4j)
+    1. [createDevon4jProject](#createDevon4jProject)
+    2. [buildJava](#buildJava)
+    4. [runServerJava](#runServerJava)
+    
+* [devon4ng Functions](#devon4ng)
+    1. [createDevon4ngProject](#createDevon4ngProject)
+    2. [buildNg](#buildNg)
+    3. [npmInstall](#npmInstall)
+    4. [runClientNg](#runClientNg)
+    
+* [Docker Functions](#dockerSpecific)
+    1. [dockerCompose](#dockerCompose)
 
-***
+* [File And Directory Functions](#fileSpecific)
+    1. [changeFile](#changeFile)
+    2. [createFile](#createFile)  
+    3. [downloadFile](#downloadFile)
+    4. [openFile](#openFile)
 
-### installDevonfwIde <a name="installDevonfwIde"></a>
-This function allows you to install devonfw-ide in your environment.
-#### parameter
-This function consist of two parameter.
+* [Katacoda Environment Functions](#katacodaSpecific)
+    1. [displayContent](#displayContent)
+    
+* [Git Functions](#gitSpecific)
+    1. [cloneRepository](#cloneRepository)
 
-1. First parameter:
-    - **Required**
-    - **Type**- String[] or Array of String
-    - **Description**-The tools you want to install within the devonfw-ide. You can pass every tool that can be installed with the devonfw-ide. In this parameter, the tools you can pass are: java, mvn, node, npm, gradle, ionic jasypt, jenkins, ng, sonar and yarn. You can pass any tool based on your requirement. 
+* [Workspace Functions](#workspaceSpecific)
+    1. [changeWorkspace](#changeWorkspace)
+    2. [createFolder](#createFolder)
```

Comment:
Move createFolder to [File And Directory Functions]

Reply:
done